src/modules/createList.js

```diff
@@ -18,12 +20,21 @@ const createList = (index) => {
   const deleteIcon = document.createElement('img');
   deleteIcon.setAttribute('src', 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABgAAAAYCAYAAADgdz34AAAAAXNSR0IArs4c6QAAAL9JREFUSEvtlc0NwjAMhV8mYQRgE07OGjABrBOfOgpsUCYxqtRKoU3tkDTi0tyi/Hzv2UqeQ+PhGt8PFRBCuDjn7gBOK0J6Ebl577s1oSqAmXsAB8Plk4jOpQAZDhJRUggzq+vDWcvBtoBJUW3jY8dfDpoDJuU5tY1davuLm7cpYK7QmsfwLAfWhdUl2gGLd/VrSfYemF9TdYksQgkgJ2jm3BcRLZIv+ZLHqHwAOFrqx/W3iFxT0fnf0M9Ur277APlk1BlhQ27RAAAAAElFTkSuQmCC');
   deleteIcon.setAttribute('alt', 'delete icon');
+  deleteIcon.className = 'deleteIcon';
 
   li.append(checkbox, input, deleteIcon);
   ul.appendChild(li);
 
   deleteIcon.addEventListener('click', () => {
-    removeList(deleteIcon, index);
+    removeList(index);
```

Comment:
- [ ] it is not possible to delete a task unless it is marked as complete. A user can change his/her mind about a particular task and decide to remove it without necessarily completing it. Please fix that